dashboard/src/modules/containers/MainLayout/index.jsx

```diff
@@ -4,6 +4,10 @@ import ToastComponent from "modules/components/ToastNotificationComponent";
 import { useSelector } from "react-redux";
 import { Spinner, Page, PageSection } from "@patternfly/react-core";
 import HeaderComponent from "modules/components/HeaderComponent";
+import NavbarDrawer from '../../components/NavbarDrawerComponent';
+import Sidebar from '../../components/SidebarComponent';
+import { TableWithFavorite } from 'modules/components/TableComponent';
```

Comment:
Does Prettier make the choice of quotation marks on the `import` statements uniform?

Also, do we still need the `../../` in lines 7 & 8, or can we start the path with `modules/`, like we do in lines 6 & 9?

Reply:
Done!